packages/ag-charts-community/src/util/object.ts

```diff
@@ -59,3 +59,16 @@ export function mergeDefaults<TSource extends Record<string, any>, TArgs extends
 
     return target as Intersection<Exclude<TArgs[number], FalsyType>>;
 }
+
+// Similar to Object.assign, but only copy an explicit set of keys.
+export function partialAssign<T>(keysToCopy: (keyof T)[], target: T, source?: Partial<T>): T {
+    if (source !== undefined) {
```

Comment:
Nit: Maybe short-circuit the `source === undefined` case up-front to remove a level of nesting? Also I would prefer using a standard `for ... of` loop over `.forEach()` if the difference is trivial, since it's both a bit more readable and less overhead?